hugo/layouts/index.html

```diff
@@ -15,6 +15,10 @@
 </script>
 {{ end }}
 
+{{ define "meta" }}
+    <link rel="alternate" type="application/rss+xml" href="{{ "/papers/index.xml" | relURL }}" title="ACL Anthology Papers" />
```

Comment:
Same here and throughout the rest of this file; input to `relURL` should not begin with a slash.

Reply:
I think I copied the URL style from the download links (e.g. `href="{{ "/anthology.bib.gz" | relURL }}"`). Should those change too, or is there a reason why they have the leading slash?

Reply:
I guess nobody noticed there because we don't generate the full BibTeX in the previews :)

Technically it should be "more correct" without the slash, so that it works in case the root is in a subdirectory, too.

Reply:
Seems plausible. :smile: I'll also update those then, just to be consistent. :+1: